src/main/cpp/hw/DragonSparkMax.cpp

```diff
@@ -191,12 +205,15 @@ bool DragonSparkMax::IsForwardLimitSwitchClosed()
 
 bool DragonSparkMax::IsReverseLimitSwitchClosed()
 {
+    if (m_spark->GetReverseLimitSwitch(m_reverseType).Get())
+        m_encoder.SetPosition(0);
```

Comment:
Do we always want to zero the encoder when the reverse limit switch is tripped?

Reply:
I think this makes sense, if you hit the bottom of some moving part, that is 0? What do you propose, move this to the mech? I am okay with that too